recipes/pymedphys_analysis/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "pymedphys_analysis" %}
+{% set version = "0.8.2" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f8e48f04c8e129dc1c96cff74a67a60ccb044d40f280c3f3b1e3520f8b7d14cd
+
+build:
+  number: 0
+  noarch: python
+  script: "python -m pip install . -vvv"
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - matplotlib
+    - numpy>=1.12
+    - pydicom
+    - pymedphys_dicom==0.8.2
```

Maybe use the following as they are all released together?
```suggestion
    - pymedphys_dicom =={{ version }}
```

---

Great idea. Yup, will do.

---

Yup, done.